### PR TITLE
Add conda-sphinx-theme to 23.10.x

### PR DESCRIPTION
**Pull request by travishathaway**
### Description

Fixes: #13300 

This is the implementation for the `23.10.x` branch which will be identical to this open pull request:

- #13298

### Checklist - did you ...

- [ ] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
- [ ] Add / update necessary tests?
- [x] Add / update outdated documentation?

### CODE REVIEW

File: docs/source/configuration.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Conda configuration
+Configuration
 ===================
```
Comment (kathatherine) [inline docs/source/configuration.rst:2]:
```suggestion
=============
```

File: docs/source/dev-guide/plugins/health_checks.rst [hunk 2]
```diff
@@ -0,0 +1,13 @@
+============
+Health-checks
+============
+
+Conda doctor can be extended with the ``health_checks`` plugin hook.
+Write new health checks using the ``health_checks`` plugin hook, install the plugins you wrote and they will run every time ``conda doctor`` command is run.
```
Comment (kathatherine) [inline docs/source/dev-guide/plugins/health_checks.rst:6]:
```suggestion
Write new health checks using the ``health_checks`` plugin hook, install the plugins you wrote, and they will run every time ``conda doctor`` command is run.
```

File: docs/source/index.rst [hunk 3]
```diff
@@ -1,53 +1,164 @@
...
-package and environment management for Python, R, Node.js, Java, and other
-application stacks. Conda is also available on
-`conda-forge <https://anaconda.org/conda-forge/conda>`_, a community channel.
-You may also get conda on `PyPI <https://pypi.org/>`_, but
-that approach may not be as up to date.
+===================
+Conda Documentation
+===================
+
+Welcome to conda's documentation! Conda provides package, dependency and environment
```
Comment (kathatherine) [inline docs/source/index.rst:5]:
```suggestion
Welcome to conda's documentation! Conda provides package, dependency, and environment
```

File: docs/source/index.rst [hunk 4]
```diff
@@ -1,53 +1,164 @@
...
+        :link: /dev-guide/contributing
+        :link-type: doc
+
+        Learn more about how the conda project is managed and how to contribute
+
+    .. grid-item-card:: Development environment :octicon:`file-code;1em;sd-text-primary`
+        :link: /dev-guide/development-environment
+        :link-type: doc
+
+        Follow this guide to get your own development environment set up.
```
Comment (kathatherine) [inline docs/source/index.rst:159]:
```suggestion
        Follow this guide to get your own development environment set up
```

File: docs/source/user-guide/index.rst [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ==========
-User guide
+User Guide
```
Comment (kathatherine) [inline docs/source/user-guide/index.rst:2]: Same capitalization question here as in the other PR.

File: docs/source/index.rst [hunk 6]
```diff
@@ -1,53 +1,164 @@
...
+        :link: /user-guide/getting-started
+        :link-type: doc
+
+        Learn the basics of using conda such as creating and adding packages to environments
+
+    .. grid-item-card:: Managing environments :octicon:`file-submodule;1em;sd-text-primary`
+        :link: /user-guide/tasks/manage-environments
+        :link-type: doc
+
+        Go in depth about environments and best practices for using them in your projects
```
Comment (kathatherine) [inline docs/source/index.rst:101]: Same comment as the other PR.

Got two "in depth"s here pretty close together. Maybe we could change the first one to "Go deeper with environments" or "Learn more about environments" or something similar.

**Review by kathatherine — APPROVED** [review]

File: docs/source/user-guide/tasks/manage-virtual.rst [hunk 7]
```diff
@@ -19,6 +15,7 @@ corresponds to the package. The currently supported list of virtual packages inc
   * ``__linux``: Available when running on Linux.
   * ``__unix``: Available when running on OSX or Linux.
   * ``__win``: Available when running on Win.
+  * ``__conda``: Version of conda that is being used for solving.
```
Comment (kenodegard) [inline docs/source/user-guide/tasks/manage-virtual.rst:18]: This is to be released in conda 23.11.0

```suggestion
```

File: docs/source/_static/switcher.json
Comment (kenodegard) [inline docs/source/_static/switcher.json:1]: Do we not need this in https://github.com/conda/conda/pull/13298?

**Review by kenodegard — CHANGES_REQUESTED** [review]

File: docs/source/user-guide/tasks/manage-virtual.rst (hunk 7 above)
Comment (travishathaway) [reply docs/source/user-guide/tasks/manage-virtual.rst:18]: Good catch!

File: docs/source/_static/switcher.json
Comment (travishathaway) [reply docs/source/_static/switcher.json:1]: Removed it!

File: docs/source/dev-guide/plugins/health_checks.rst
Comment (kenodegard) [inline docs/source/dev-guide/plugins/health_checks.rst:1]: Also not released yet (https://github.com/conda/conda/commit/c715853d48465c4f181d9a00051a53e52b0ec8f7)

**Review by kenodegard — APPROVED** [review]

**Comment by beeankha** [comment]
The same issue [I mentioned](https://github.com/conda/conda/pull/13298#pullrequestreview-1753825392) in https://github.com/conda/conda/pull/13298 is showing up when I build these docs locally, though I'm not sure if it's something on my end or something that's fixable via this PR:

Screenshot 2023-11-28 at 2 55 08 PM

**Pull request merged**